ArduPlane/ArduPlane.cpp

```diff
@@ -435,7 +438,7 @@ void Plane::airspeed_ratio_update(void)
         // don't calibrate when going beyond normal flight envelope
         return;
     }
-    const Vector3f &vg = gps.velocity();
+//    const Vector3f &vg = gps.velocity();
```

Comment:
please remove instead of commenting out

Reply:
that patch shouldn't be merged - it was just a hack to allow for displaying of raw airspeed on ground for debugging

Reply:
Sorry, I should have mentioned that that commit has to come out before merging